lib/Screens/login_page.dart

```diff
@@ -0,0 +1,133 @@
+                width: MediaQuery.of(context).size.height * 0.4,
+                height: MediaQuery.of(context).size.height * 0.7,
+                decoration: BoxDecoration(
+                  color: Colors.white,
+                  borderRadius: BorderRadius.circular(20),
+                ),
+                child: Padding(
+                  padding: const EdgeInsets.symmetric(horizontal: 75),
+                child: Center(
+                  child: Column(
+                    children: [
+                      const SizedBox(
+                        height: 15,
+                      ),
+                      const Image(
+                          image: AssetImage('images/3898671.png')),
+                      const Text(
+                        "Login",
+                        style: TextStyle(
+                            color: Color.fromRGBO(170, 126, 189, 100),
+                            fontWeight: FontWeight.bold,
+                            fontSize: 24),
+                      ),
+                      const SizedBox(
+                        height: 45,
+                      ),
+                      Padding(padding: const EdgeInsets.symmetric(horizontal: 50,vertical: 20),
+                      child:TextField(
+                        decoration: const InputDecoration(
+                        border: InputBorder.none,
+                        hintText: "Username",
+                        hintStyle: TextStyle(color: Colors.grey),
+                      ),
+                  controller: usernameController,
+                ),
+                ),
+                  Padding(padding: const EdgeInsets.symmetric(horizontal: 50,vertical: 20),
+                      child: TextField(
+                  keyboardType: TextInputType.visiblePassword,
+                  decoration: const InputDecoration(
+                    border: InputBorder.none,
+                    hintText: "Password",
+                    hintStyle: TextStyle(color: Colors.grey),
+                  ),
+                  controller: passwordController,
+                ),
+                ),
+                      Padding(padding: const EdgeInsets.all(20),
+                      child:Row(
+                          children: [
+                          loginButton(),
+                    ],
+                  )
+                      ),
+
+                    ],
+                  ),
+                ),
+                ),
+              ),
+          ),
+          ),
+              ],
+            )
+          ),
+    ),
+    );
+  }
+
+  Widget loginButton(){
+    return ElevatedButton(
+        style: ElevatedButton.styleFrom(
+          padding: const EdgeInsets.symmetric(horizontal: 50, vertical: 20),
+          backgroundColor: const Color.fromRGBO(170, 126, 189, 100)
+        ),
+        onPressed :() async {
+      Auth.setUsername(usernameController.text);
+      Auth.setPassword(passwordController.text);
+
+      await widget.api.fetchToken();
```

Comment:
- No error is shown when login fails. Show a dialog or snack bar to indicate the user the result of his/her action